fe/fe-core/src/main/java/com/starrocks/analysis/MultiRangePartitionDesc.java

```diff
@@ -127,6 +133,22 @@ public void setTimeUnit(String timeUnit) {
         List<SingleRangePartitionDesc> singleRangePartitionDescs = Lists.newArrayList();
         long currentLoopNum = 0;
         long maxAllowedLimit = Config.max_partitions_in_one_batch;
+
+        // By default, the dayOfWeek parameter are the same as those of dynamic partition.
```

Comment:
1. The problem is to make batching partition semantics is consistent with dynamic partition.
    In china, the Monday is the first day of week. In western country, the Sunday is the first day of week. The semantics is should be consistent between batching partition and dynamic partition. If the option is not set, the Monday will be the first day of week.
2. The 1st January is the first week of every year. Every year have 52 weeks. The last week will end at 31st December.

Reply:
> The 1st January is the first week of every year. Every year have 52 weeks. The last week will end at 31st December.

+1